src/example.py

```diff
@@ -21,36 +26,42 @@
 }
 
 def benchmark(src: str):
-    from cProfile import Profile
-    from io import StringIO
-    from contextlib import redirect_stdout
+    print('benchmarking: ')
+    print('\t', src)
 
     pr = Profile()
     pr.enable()
     # sort by total time executing in a function's body (not including sub-calls)
     pr.run(src)
     pr.disable()
+    print()
     
     # don't allow benchmarking to dump in stdio
     f = StringIO()
     with redirect_stdout(f):
         pr.print_stats(sort='tottime')
     
     # truncate stdout to a dozen lines
-    print('\n'.join(f.getvalue().splitlines()[:12]))
-
-
-# test code:
-#   synthesize(input_gdfs, target)
-# completed in:
-#   without dataclass optimization: ~9.1 secs
-#   with    dataclass optimization: ~8.9 secs
-#   with    dataclass and no print: ~8.8 secs
-# benchmark('synthesize(input_gdfs, target)')
-
-# test code:
-#   lazy_synthesize(input_gdfs, target)
-# completed in:
-#   without dataclass optimization: ~0.36 secs
-#   with    dataclass optimization: ~0.35 secs
-benchmark('lazy_synthesize(input_gdfs, target)')
+    print('\n'.join(f.getvalue().splitlines()[:8]))
```

Comment:
Nit — let's update the comment above since we truncate to 8 lines here!

Reply:
Got it!